OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -60,7 +60,7 @@ public static bool ValidateCommand(S server, Connection conn, Session.Client cli
 
 		static void CheckAutoStart(S server)
 		{
-			var playerClients = server.LobbyInfo.Clients.Where(c => c.Bot == null && c.Slot != null);
+			var playerClients = server.LobbyInfo.Clients.Where(c => (c.Bot == null && c.Slot != null) || c.IsAdmin);
```

Comment:
Can you please add a comment here explaining that spectating admins are an exception to the normal rules?